### DISCUSSION TITLE
Page action dropdown

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#3619] [Help] [answered]
Hi,

Is it possible to have page actions as a dropdown, just like we have for table row actions?
I have many buttons on a page and the UI feels a bit congested.

Thanks

**Comment by danharrin** [comment, marked as answer]
Yes, you can wrap them in an `ActionGroup`

**Reply by jwhoami** [reply]
Perfect. works as expected.
And even documented https://filamentphp.com/docs/2.x/admin/pages/actions#grouping
Thanks
